subsys/logging/log_output.c

```diff
@@ -203,94 +203,83 @@ static void std_print(struct log_msg *msg,
 		      const struct log_output *log_output)
 {
 	const char *str = log_msg_str_get(msg);
+	u32_t nargs = log_msg_nargs_get(msg);
+	u32_t args[nargs];
```

Comment:
The VLA usually translates to malloc(). Please use alloca() here.

Reply:
done